login_test.go

```diff
@@ -19,10 +20,21 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+var loginTestConfiguration *config.ConfigurationData
+
+func init() {
+	var err error
+	loginTestConfiguration, err = config.GetConfigurationData()
+	if err != nil {
+		panic(fmt.Errorf("Failed to setup the configuration: %s", err.Error()))
+	}
+}
+
 func newTestKeycloakOAuthProvider() *login.KeycloakOAuthProvider {
+
 	oauth := &oauth2.Config{
-		ClientID:     configuration.GetKeycloakClientID(),
-		ClientSecret: configuration.GetKeycloakSecret(),
+		ClientID:     loginTestConfiguration.GetKeycloakClientID(),
+		ClientSecret: loginTestConfiguration.GetKeycloakSecret(),
 		Scopes:       []string{"user:email"},
 		Endpoint: oauth2.Endpoint{
 			AuthURL:  "http://sso.demo.almighty.io/auth/realms/fabric8/protocol/openid-connect/auth",
```

Comment:
As I mentioned above no need to hard code the endpoin URLs in the tests anymore. They will be set up in the service.